packages/relay-compiler/bin/RelayCompilerBin.js

```diff
@@ -122,6 +128,9 @@ function getRelayFileWriter(baseDir: string) {
 function getSchema(schemaPath: string): GraphQLSchema {
   try {
     let source = fs.readFileSync(schemaPath, 'utf8');
+    if (path.extname(schemaPath) === '.json') {
+      source = printSchema(buildClientSchema(JSON.parse(source).data));
+    }
```

Comment:
Do you want me to leave a todo?
@leebyron What about my proposal to add support for GraphQL endpoint? Thanks